src/boot/efi/boot.c

```diff
@@ -367,139 +367,118 @@ static UINTN entry_lookup_key(Config *config, UINTN start, CHAR16 key) {
         return -1;
 }
 
+#define PS_FORMAT(prop, fmt, ...) Print(L"%*s: " fmt L"\n", padding, prop, ## __VA_ARGS__)
+#define PS_STRING(prop, str) if (str) { Print(L"%*s: %s\n", padding, prop, str); }
+#define PS_BOOL(prop, value) PS_STRING(prop, yes_no(value))
```

Comment:
why as macros? can do as static functions too, no?

i mean, macros are fine sometimes, but if we can avoid them, i'd always avoid it.

Reply:
This macro expects padding to be in scope (hence why I #undef it later). Sure, this can be functions, but then we want a global var for padding, which feels kinda silly for this use case.... And you don't like nested functions...

Reply:
i don't think the padding arg is that bad though, no?